rust/sealevel/client/src/context.rs

```diff
@@ -183,30 +188,46 @@ impl<'ctx, 'rpc> TxnBuilder<'ctx, 'rpc> {
 
     pub(crate) fn send_with_payer(self) -> Option<EncodedConfirmedTransactionWithStatusMeta> {
         let payer_signer = self.ctx.payer_signer();
-        self.send(&[&*payer_signer])
+        let payer_pubkey = self.ctx.payer_pubkey;
+        self.send(&[payer_signer.as_deref()], &payer_pubkey)
     }
 
-    pub(crate) fn send<T: Signers>(
+    /// Sends the transaction with a signer for the given pubkey.
+    /// Note that a pubkey may not have an associated keypair, in which case
+    /// this function will return None and transactions will be printed to stdout
+    /// for user confirmation & manual submission.
```

Comment:
it looks like `NullSigner` is no longer being created in `payer_signer` after these changes

Reply:
yeah, I believe the current version removed that comment if that's what this was calling out